RedditSharp/WebAgent.cs

```diff
@@ -104,8 +104,7 @@ static WebAgent()
 
         public virtual JToken CreateAndExecuteRequest(string url)
         {
-            Uri uri;
-            if (!Uri.TryCreate(url, UriKind.Absolute, out uri))
+            if (!Uri.TryCreate(url, UriKind.Absolute, out var uri))
```

Comment:
I second this, the slightly more wordy version is much more readable to me and feels more specific/easier to follow.